projects/plugins/protect/src/js/components/footer/index.jsx

```diff
@@ -0,0 +1,51 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+import { Container, Col, Text, Button } from '@automattic/jetpack-components';
+import { __ } from '@wordpress/i18n';
+import { Icon, bug, cloud, shield } from '@wordpress/icons';
```

Comment:
```suggestion
import { Icon, bug, cloud, shield, Title } from '@wordpress/icons';
```

because of the following comments below.